src/databricks/labs/ucx/assessment/queries/02_0_database_summary.sql

```diff
@@ -0,0 +1,53 @@
+-- viz type=table, name=Database Summary, search_by=database, columns=database,upgrade,tables,views,dbfs_root,delta_tables,total_grants,granted_principals,database_grants,table_grants,service_principal_grants,user_grants,group_grants
+-- widget title=Database Summary, col=0, row=2, size_x=6, size_y=8
+WITH table_stats AS (
+  SELECT
+    `database`,
+    object_type,
+    UPPER(table_format) AS `format`,
+    `location`,
+    IF(object_type IN ("MANAGED", "EXTERNAL"), 1, 0) AS is_table,
+    IF(object_type = "VIEW", 1, 0) AS is_view,
+    CASE
+        WHEN STARTSWITH(location, "dbfs:/") AND NOT STARTSWITH(location, "dbfs:/mnt") THEN 1
+        WHEN STARTSWITH(location, "/dbfs/") AND NOT STARTSWITH(location, "/dbfs/mnt") THEN 1
+        ELSE 0
+    END AS is_dbfs_root,
+    CASE
+        WHEN STARTSWITH(location, "wasb") THEN 1
+        WHEN STARTSWITH(location, "adl") THEN 1
+        ELSE 0
+    END AS is_unsupported,
+    IF(format = "DELTA", 1, 0) AS is_delta
+   FROM $inventory.tables
+), database_stats AS (
+  SELECT `database`,
+    CASE
+        WHEN SUM(is_table) = 0 AND SUM(is_view) > 0 THEN "View Migration Required"
+        WHEN SUM(is_dbfs_root)/SUM(is_table) > .3 THEN "Asset Replication Required"
+        WHEN SUM(is_delta)/SUM(is_table) < .7 THEN "Some Non Delta Assets"
+        WHEN SUM(is_unsupported)/SUM(is_table) > .7 THEN "Storage Migration Required"
+        ELSE "In Place Sync"
+    END AS upgrade,
+    SUM(is_table) AS tables,
+    SUM(is_view) AS views,
+    SUM(is_unsupported) AS unsupported,
```

Comment:
This is not the same as the `columns=` list above. I see `unsupported` missing there. I trust this is OK and works like this?